todo_list/views/views_task.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import render, redirect, get_object_or_404
+
+from forms import TaskForm
+from todo_list.models import Task
+
+
+def create_task_view(request, page_id=0):
+    if request.method == 'GET':
+        form = TaskForm()
+        return render(request, 'todo_list/task/task_create.html', {'form': form})
+
+    else:
+        data = request.POST.copy()
+        data['page'] = page_id  # set FK
```

Comment:
@quratulain25 You may define a hidden field on frontend form and utilize that to pass FK.